src/ansi-c/goto-conversion/goto_clean_expr.cpp

```diff
@@ -151,9 +170,13 @@ static exprt convert_statement_expression(
     auto &current_it = paths.back_it();
     auto &path_condition = paths.back_path_condition();
     auto &value_map = paths.back_value_map();
-    INVARIANT(
-      current_it != where.instructions.end(),
-      "Quantifier body must have a unique end expression.");
+
+    if(current_it == where.instructions.end())
+    {
+      res = and_exprt(res, implies_exprt(path_condition, true_exprt()));
```

Comment:
Why are you constructing the expression in exactly this way? `implies_exprt(path_condition, true_exprt())` is semantically equivalent to `true_exprt()` after all?!

Reply:
Yes, you are right. I will fix it.